WreckEngine/Source.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
I will not budge on `#pragma once` lines. They protect against user and compiler stupidity.